core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -120,6 +124,13 @@ public final synchronized void start(Listener listener) {
     executor = SharedResourceHolder.get(executorResource);
     this.listener = Preconditions.checkNotNull(listener, "listener");
     resolve();
+
+    Long addressTtl = Longs.tryParse(Strings.nullToEmpty(
+          java.security.Security.getProperty("networkaddress.cache.ttl")));
```

Comment:
Hmm... so if it is the default then we don't poll? The default varies, but is 30 seconds in many cases which means that polling would be useful (although I'm not saying polling every 30 seconds is what we want).